app/src/androidTest/java/com/skydoves/pokedex/DetailActivityInjectionTest.kt

```diff
@@ -49,9 +50,13 @@ class DetailActivityInjectionTest {
       it.moveToState(Lifecycle.State.CREATED)
       it.onActivity { activity ->
         assertThat(activity.viewModel).isNotNull()
-        activity.viewModel.pokemonInfoLiveData.observe(activity) { pokemonList ->
-          assertThat(pokemonList).isNotNull()
-        }
+
+        activity.viewModel.pokemonInfoLiveData.observe(
+          activity,
+          Observer { pokemonList ->
+            assertThat(pokemonList).isNotNull()
+          }
+        )
```

Comment:
Is there any reason to use Observer? 
In kotlin 1.4.0, we can use SAM conversion and we don't need to declare a single abstract method in the trailing lambda.

Reply:
oops, it was error in my local before. Then I update kotlin plugin to the latest and the error gone, I'll revert back